packages/dashboard/app/components/AgentsView.css

```
@@ -54,6 +54,9 @@
   flex-direction: column;
   flex: 1;
```

Stylelint (17.12.0)

[error] Duplicate property "flex" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)